src/Phinx/Console/Command/Create.php

```diff
@@ -106,15 +119,54 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $fileName = Util::mapClassNameToFileName($className);
         $filePath = $path . DIRECTORY_SEPARATOR . $fileName;
 
-        if (file_exists($filePath)) {
+        if (is_file($filePath)) {
             throw new \InvalidArgumentException(sprintf(
                 'The file "%s" already exists',
                 $filePath
             ));
         }
 
-        // load the migration template
-        $contents = $this->getMigrationTemplate();
+        // Get the alternative template and static class options, but only allow one of them.
+        $altTemplate = $input->getOption('template');
+        $creationClassName = $input->getOption('class');
+        if ($altTemplate && $creationClassName) {
+            throw new \InvalidArgumentException('Cannot use --template and --class');
```

Comment:
The wording here could be improved, it should clarify that the two cannot be used together:

> Cannot use --template and --class at the same time.